### ISSUE TITLE
Why  is training accuracy  always  less than test accuracy in deep retrieval model  ? 

### ISSUE THREAD

**Issue by AzizIlyosov** [tensorflow/recommenders#419]
I followed [this ](https://www.tensorflow.org/recommenders/examples/deep_recommenders)tutorial to train movielens dataset. But one thing I noticed  here is  that, training accuracy is  not changing during all epochs(it  is always  zero ), although test accuracy is fluctuating. Does it  mean model is underfitting? If  yes  how can I solve this problem  ?

**Comment by muhajirakbarhsb** [comment]
you can replace two towers with parameter user for candidate and query tower

**Comment by maciejkula** [comment]
This is because computing the accuracy during training is disabled (see `compute_metrics=not training`) in order to speed up training.

Changing this to `True` will compute the accuracy during training as well.

**Comment by AzizIlyosov** [comment]
Thanks  a lot  !
